### PR TITLE
Add vollseg recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@kapoorlab, do you agree with being added as a maintainer?

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17766
- https://github.com/conda-forge/staged-recipes/pull/17845

**Comment by kapoorlab** [comment]
@goanpeca Yes I agree on being added as the maintainer.

**Comment by kapoorlab** [comment]
@conda-forge-admin, please add bot automerge[¶](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-bot-automerge)

**Comment by kapoorlab** [comment]
@conda-forge-admin, please rerun bot[¶](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-rerun-bot)

**Comment by jaimergp** [comment]
Hi @kapoorlab! We need to wait for https://github.com/conda-forge/staged-recipes/pull/17845 to be merged first, since it's a dependency here.

**Comment by jaimergp** [comment]
@kapoorlab This is now more or less there. I am checking the README though and I see that Tensorflow cannot be above 2.3.4. Is that still accurate? conda-forge only has tensorflow 2.4 and above.

File: recipes/vollseg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - diplib-python
+    - imagecodecs
+    - matplotlib-base
+    - napari
+    - opencv
+    - pandas
+    - scipy
+    - stardist >=0.7.0
+    - tifffile
+    - csbdeep
```
Comment (jaimergp) [inline recipes/vollseg/meta.yaml:32]:
```suggestion
    - csbdeep
    - tensorflow
```

**Pull request merged**